tests/rest_api/test_get_collection.py

```diff
@@ -0,0 +1,20 @@
+import requests
+import pytest
+
+BASE_URL = "https://drb-api-qa.nypl.org"
+
+def test_get_collection():
+    url = BASE_URL + "/collection/3650664c-c8be-4d07-8d64-2d7003b02048"
+    expected_status_code = 200
+    response = requests.get(url)
+    assert expected_status_code is not None
```

Comment:
Should we be asserting that expected_status_code is not None? this is declared above.

Reply:
yeah my bad on this.